### DISCUSSION TITLE
Most used addons since v1.21.0 according to feedback messages

### DISCUSSION THREAD

**Discussion by WorldLanguages** [ScratchAddons/ScratchAddons#4100] [Discussion]
According to ~250 feedback messages. I didn't remove duplicates this time because I forgot.
Like last time, take these numbers with a grain of salt.
I wouldn't recommend looking at the addons released after 1.19.0 since they have an advantage or a disadvantage, depending on how you look at it.

![](https://cdn.discordapp.com/attachments/755073703393624177/934198824749117500/sa-feedback-chart.png)

**Comment by lisa-wolfgang** [comment]
~~Just do involuntary analytics~~
